gems/sorbet-runtime/lib/types/private/methods/decl_builder.rb

```diff
@@ -249,7 +249,7 @@ def finalize!
         decl.bind = nil
       end
       if decl.checked.equal?(ARG_NOT_PROVIDED)
-        decl.checked = :always
+        decl.checked = T::Private::RuntimeLevels.default_checked_level
```

Comment:
As currently implemented,

```ruby
sig {returns(...).soft(...).checked(:never)}
```

is an error, but

```ruby
T::Configuration.default_checked_level = :never
sig {returns(...).soft}
```

is not an error, but the `.soft` will be ignored. This is probably a bug, and I think i'd like to fix it.

Reply:
This invariant is easier to enforce on my other branch that re-enables .checked. I'll add it there.